### PR TITLE
Dependency alignment: create a config file with SMTP URL

### PR DESCRIPTION
**Pull request by TomasHofman**
Adding a rule that should create a parametrized settings file for dependency alignment script.

Cc @rpelisse @spyrkob

### CODE REVIEW

**Comment by TomasHofman** [comment]
Related harmonia PR: https://github.com/jboss-set/harmonia/pull/54

File: roles/component_alignment/tasks/main.yml [hunk 1]
```diff
@@ -49,3 +49,11 @@
 
 - include_tasks: tasks/deploy_config.yml
   loop: "{{ component_alignment.config.files }}"
+
```
Comment (rpelisse) [inline roles/component_alignment/tasks/main.yml:52]: @TomasHofman just to be super clean, I would add to the assert: block that check  mailer,  mailer.smtp,  mailer.smtp.host and  mailer.smtp.port needs to be defined. Other than that, it looks good to me.  

@spyrkob We may want to rename this /opt/tools/component-alignment-settings.sh into something like /opt/tools/send_email.sh and have other "mail sending things" use it.

**Review by rpelisse — CHANGES_REQUESTED** [review]

File: roles/component_alignment/tasks/main.yml (hunk 1 above)
Comment (TomasHofman) [reply roles/component_alignment/tasks/main.yml:52]: So I added the assert (please check) and renamed the file to `smtp-settings.sh` right away, so I can sync the harmonia PR. You might still want to move the task to different role though...

**Comment by spyrkob** [comment]
I think we can merge it now and refactor later (either when we need to use it outside of comp-alligment or during Da Big Refactoring)

**Review by rpelisse — APPROVED** [review]

File: roles/component_alignment/tasks/main.yml (hunk 1 above)
Comment (TomasHofman) [reply roles/component_alignment/tasks/main.yml:56]: Moved the assertions to the already existing assert block at the beginning of the file.

**Pull request merged**